test/resources/microshift-rpm.resource

```diff
@@ -27,12 +27,19 @@ Install MicroShift RPM Packages From System Repo
 
 Install MicroShift RPM Packages From Repo
     [Documentation]    Installs MicroShift RPM packages from the specified URL
-    [Arguments]    ${repo_url}    ${version}
-    Configure MicroShift Repository    ${repo_url}
-    Install MicroShift RPM Packages From System Repo    ${version}
-    Unconfigure MicroShift Repository
+    [Arguments]    ${repo}    ${version}
 
-Configure MicroShift Repository
+    IF    "http" in "${repo}"
+        Configure MicroShift File Repository    ${repo}
+        Install MicroShift RPM Packages From System Repo    ${version}
+        Unconfigure MicroShift File Repository
+    ELSE
+        Command Should Work    subscription-manager repos --enable ${repo}
+        Install MicroShift RPM Packages From System Repo    ${version}
+        Command Should Work    subscription-manager repos --disable ${repo}
```

Comment:
There's a risk that this would disable repo with dependencies (e.g. 4.15 at 4.16/main)